server/sonar-server/src/main/java/org/sonar/server/settings/ws/ValuesAction.java

```diff
@@ -211,6 +208,14 @@ private void processSettings() {
       return valueBuilder;
     }
 
+    private void setInherited(Setting setting, Settings.Setting.Builder valueBuilder) {
+      if (setting.isDefault()) {
```

Comment:
suggestion: a boolean `isInherited` and then `valueBuilder.setInherited(isInherited)`. In case it makes things clearer